### PR TITLE
panel_2

### LINKED ISSUE #186: panel_2
panel_2

NCL script: https://www.ncl.ucar.edu/Applications/Scripts/panel_2.ncl
NCL plot: https://www.ncl.ucar.edu/Applications/Images/panel_2_lg.png

Category: Panel

### PR DESCRIPTION
**Pull request by anissa111**
Closes #186 

NCL Original:
image

Python generated:
image

### CODE REVIEW

File: Plots/Panels/NCL_panel_2.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import xarray as xr
+import matplotlib.pyplot as plt
+import cartopy
+import cartopy.crs as ccrs
+from cartopy.mpl.gridliner import LongitudeFormatter, LatitudeFormatter
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_2.py:21]:
```suggestion

```
Unused line of code

File: Plots/Panels/NCL_panel_2.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    levels=levels,  # contour levels specified outside this function
+    xticks=np.arange(-180, 181, 30),  # nice x ticks
+    yticks=np.arange(-90, 91, 30),  # nice y ticks
+    transform=projection,  # ds projection
+    add_colorbar=False,  # don't add individual colorbars for each plot call
+    add_labels=False,  # turn off xarray's automatic Lat, lon labels
+    colors="k",  # note plurals in this and following kwargs
+    linestyles="-",
+    linewidths=0.5,
+)
```
Comment (michaelavs) [inline Plots/Panels/NCL_panel_2.py:70]: I would put this parenthesis on line 69 and remove the additional comma after 0.5 for formatting consistency

Comment (anissa111) [reply Plots/Panels/NCL_panel_2.py:69]: Done!

**Comment by anissa111** [comment]
Original:
image

Python generated:
image

**Pull request merged**